srcpkgs/vaultwarden/patches/dashmap_available_parallelism.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/Cargo.toml b/Cargo.toml
+index 1ccd16c..5d0dab6 100644
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -155,6 +155,9 @@ mimalloc = { version = "0.1.29", features = ["secure"], default-features = false
+ # 2022-05-04: Forked/Updated the job_scheduler again use the latest dependencies and some fixes.
+ job_scheduler = { git = 'https://github.com/BlackDex/job_scheduler', rev = '9100fc596a083fd9c0b560f8f11f108e0a19d07e' }
+ 
++# revert a commit that breaks Rust 1.57 support
```

Comment:
instead of using a fork of the dependency, maybe it could be done like other rust packages with dependency patches:
https://github.com/void-linux/void-packages/blob/a46c737fdd1ea39c6ea9f8fd520d0715f2d03b5c/srcpkgs/maturin/template#L25-L29